src/pages/api/signIn.ts

```diff
@@ -2,6 +2,29 @@ import { NextApiResponse } from 'next';
 import { NextApiRequest } from 'next';
 import SessionController from 'src/modules/sessions/controller/session.controller';
 
+
+/**
+ * @swagger
```

Comment:
Could you update `README.md` to show us how to use this lib? Or you could write some note in https://capstone-alumni.atlassian.net/wiki/spaces/AC/pages/393511/Technical+Investigation

Reply:
ok, I have done the guideline and report to team